teamcity/datadog_checks/teamcity/teamcity_rest.py

```diff
@@ -107,7 +107,7 @@ def _get_last_build_id(self, project_id, build_config_id):
                 self.bc_store.set_last_build_id(project_id, build_config_id, last_build_id)
 
     def _initialize_multi_build_config(self):
-        filtered_projects = None
+        filtered_projects = {}
         build_configs_list = None
```

Comment:
```suggestion
```

Not sure that's needed to declare this one here

Reply:
Good catch! I applied this locally to make sure tests pass.